app/renderer/components/tabs/content/closeTabIcon.js

```diff
@@ -49,33 +48,33 @@ class CloseTabIcon extends React.Component {
 
   mergeProps (state, ownProps) {
     const currentWindow = state.get('currentWindow')
-    const frameKey = ownProps.frameKey
-    const frame = frameStateUtil.getFrameByKey(currentWindow, frameKey) || Immutable.Map()
-    const tabId = frame.get('tabId', tabState.TAB_ID_NONE)
-    const isPinnedTab = tabState.isTabPinned(state, tabId)
+    const tabId = ownProps.tabId
+    const frameKey = frameStateUtil.getFrameKeyByTabId(currentWindow, tabId)
+    const isPinned = tabState.isTabPinned(state, tabId)
 
     const props = {}
-    // used in renderer
-    props.showCloseIcon = !isPinnedTab &&
-      (
-        closeState.deprecatedHasRelativeCloseIcon(currentWindow, frameKey) ||
-        closeState.deprecatedHasFixedCloseIcon(currentWindow, frameKey)
-      )
-
-    // used in functions
-    props.frameKey = frameKey
+    props.isPinned = isPinned
     props.fixTabWidth = ownProps.fixTabWidth
+    props.hasFrame = frameStateUtil.hasFrame(currentWindow, frameKey)
+    props.centralizeTabIcons = tabUIState.centralizeTabIcons(currentWindow, frameKey, isPinned)
+    props.showCloseIcon = closeState.showCloseTabIcon(currentWindow, frameKey)
     props.tabId = tabId
-    props.hasFrame = !frame.isEmpty()
 
     return props
   }
 
   render () {
+    if (this.props.isPinned || !this.props.showCloseIcon) {
+      return null
+    }
+
     return <TabIcon
       data-test-id='closeTabIcon'
       data-test2-id={this.props.showCloseIcon ? 'close-icon-on' : 'close-icon-off'}
-      className={css(this.props.showCloseIcon && styles.closeTab)}
+      className={css(
+        styles.closeTab__icon,
+        this.props.centralizeTabIcons && styles.closeTab__icon__centered
```

Comment:
Shouldn't it be `closeTab__icon_centered`?

Reply:
Addressed on https://github.com/brave/browser-laptop/pull/10691/files#diff-08a3077ca08d52dfaad29de7ae4d904aR76

On the commit: https://github.com/brave/browser-laptop/commit/8a1a150a2777d17b58c52895bf08b7b8a2c33c16#diff-08a3077ca08d52dfaad29de7ae4d904aR76